src/Orleans.Transactions/State/TransactionQueue.cs

```diff
@@ -328,6 +333,9 @@ public async Task NotifyOfConfirm(Guid transactionId, DateTime timeStamp)
 
         public void NotifyOfCancel(Guid transactionId, DateTime timeStamp, TransactionalStatus status)
         {
+            if (logger.IsEnabled(LogLevel.Trace))
+                logger.Trace("NotifyOfConfirm. TransactionId: {TransactionId}, TimeStamp: {TimeStamp} Status: {TransactionalStatus}", transactionId, timeStamp, status);
```

Comment:
method name in trace string does not match... you probably meant "NotifyOfCancel". Perhaps use {nameof(NotifyOfConfirm)} to ensure trace message stays in sync with method name?